src/main/scala/Game.scala

```diff
@@ -0,0 +1,6 @@
+case class Game (
```

Comment:
You might want to create a package for your various models - e.g. `scrabble.model` - and stick `Game`, `Player`, `Tile` and `Bag` in it as an easy way to move around them. Or, if your models don't have too much extra logic around them then, you could just have a file called `Models` and stick them all in there.